### DISCUSSION TITLE
Support for a new board

### DISCUSSION THREAD

**Discussion by landswellsong** [Freescale/meta-freescale#1849] [Q&A] [answered]
Hi,
I'm pretty new to the NXP ecosystem. I'm making my Yocto project run on a [new IMX8MP SoM by Olimex](https://www.olimex.com/Products/SOM/NXP-iMX8/). Once I get it stable and working is it a good idea to contribute the machine and patches here into this repository? I tried to eveasdrop on what others are doing and seems like PhyTec, Variscite, Debix etc make their own layers, so I don't know what is the proper etiquette here, should I do a PR here or should I make a separate layer and give it to Olimex to host (since they don't use Yocto for any of their boards yet)?

**Comment by angolini** [comment, marked as answer]
You should use `meta-freescale-3rdparty` for that PR

**Reply by landswellsong** [reply]
Awesome, thanks for the explanation!
